upt_macports/upt_macports.py

```diff
@@ -13,7 +15,41 @@ def __init__(self):
     def create_package(self, upt_pkg, output):
         self.upt_pkg = upt_pkg
         self.logger.info(f'Hello, creating the package')
-        print(self._render_makefile_template())
+        if output is None:
+            print(self._render_makefile_template())
+        else:
+            self._create_output_directories(upt_pkg, output)
+            self.logger.info('Directory structure created')
```

Comment:
This does not give enough info to the user. We should tell them "Created /tmp/foo/bar".